tests/sentry/tasks/deletion/test_hybrid_cloud.py

```diff
@@ -170,3 +176,132 @@ def test_control_processing(task_runner):
 
         # Do not process
         assert results.exists()
+
+
+@django_db_all
+def xtest_hybrid_cloud_foreign_keys_generate_outboxes():
```

Comment:
```suggestion
def test_hybrid_cloud_foreign_keys_generate_outboxes():
```

I don't think you want this `x` here.